Change README (#118)
arijit79/minus: README.md

</p>

## Features

> - Send data as well as configure the pager on the fly.
>     This means that your data can be shown on the pager's screen as soon as it is loaded by your application. But not only that,
>     you can also configure the minus while its running.

Author: ```suggestion
- Send data and configure the pager while it's running
```

Reviewer: What's the problem with that para?

Author: it's in my opinion too long. I wouldn't fully read the paragraph and this sentence includes all necessary information in my opinion

Reviewer: Ah! Actually GitHub is merging the lines, I actually wanted them to be separate paragraphs. Like the first line should be as it is and following lines are a bit indented. Tomorrow I will see if I can get that or just remove that entire.

- Supports separate modes for dynamic and static output display
    This separation of modes allows us to do some cool tricks in static mode. For example in static mode, if the terminal has 
    enough rows to display all the data at once then minus won't even start the pager and write all the data to the screen and quit. 

>     (Ofcourse this behaviour can be avoided if you don't like it).

Author: ```suggestion
    (Of course this behaviour can be avoided if you don't like it).
```
